src/main/java/uk/gov/hmcts/divorce/document/content/DraftApplicationTemplateContent.java

```diff
@@ -137,11 +138,9 @@ public Map<String, Object> apply(final CaseData caseData, final Long ccdCaseRefe
             templateContent.put(RESPONDENT_SOLICITOR_EMAIL,
                 isNull(applicant2Solicitor.getEmail()) ? NOT_GIVEN : applicant2Solicitor.getEmail());
             templateContent.put(RESPONDENT_SOLICITOR_FIRM_NAME,
-                isNull(applicant2Solicitor.getOrganisationPolicy())
-                    ? NOT_GIVEN
-                    : applicant2Solicitor.getOrganisationPolicy().getOrganisation().getOrganisationName());
+                StringUtils.isEmpty(applicant2Solicitor.getPreferredFirmName()) ? NOT_GIVEN : applicant2Solicitor.getPreferredFirmName());
```

Comment:
Looking at the logic of the `getPreferredFirmName`, I believe this will never be empty as it will either return `firmName` or `OrganisationName`

Reply:
It will be empty if there is no firmName and no Organisation (offline solicitor)